lib/preprocess/IConfigPreprocessor.ts

```diff
@@ -1,5 +1,10 @@
 import type { Resource } from 'rdf-object';
 
+export interface IConfigPreprocessorTransform {
+  finished: boolean;
```

Comment:
Perhaps something like `finishTransformation` might be a bit more descriptive.

Could you also add a sentence as javadoc on what it does?

Reply:
Could you also rename this field to `finishTransformation`?